pkg/runners/zero/zero_benchmark_test.go

```diff
@@ -214,12 +217,22 @@ func BenchmarkRunnerWithFibonacci(b *testing.B) {
     `)
 
 	for i := 0; i < b.N; i++ {
-		program, err := LoadCairoZeroProgram(compiledJson)
+        cairoZeroJson, err := zero.ZeroProgramFromJSON(compiledJson)
```

Comment:
I think it might be a good idea to reset the timer for this benchmark test since we already agreed with that convention.